addons/l10n_jp/data/account.tax.group.csv

```diff
@@ -1,4 +1,4 @@
 id,name,country_id/id
-tax_group_0,税対象外/免除,base.jp
-tax_group_8,消費税 8%,base.jp
-tax_group_10,消費税 10%,base.jp
+tax_group_0,Not subject to tax/exempt,base.jp
+tax_group_8,VAT 8%,base.jp
+tax_group_10,VAT 10%,base.jp
```

Comment:
You're using a lot of different terms for the same things
![image](https://user-images.githubusercontent.com/66360871/217256549-454c9bf6-b5aa-4663-8cf4-186f87cb756d.png)

Here VAT should be changed I think (consumption tax). You could also remove the (excl), (incl)